website/docs/contributing.md

```diff
@@ -0,0 +1,35 @@
+---
+sidebar_position: 3
+---
+
+# How to contribute to FerretDB
+
+FerretDB is an open source project and everyone – developers and non-developers – is welcome to contribute.
+If you’re interested in contributing to FerretDB projects, this documentation will help you get started.
+
+Here are some of the current FerretDB projects that you can fork and contribute to:
+
+* [FerretDB](https://github.com/FerretDB/FerretDB): This repository contains the FerretDB project – an open source alternative project to MongoDB.
+* [github-actions](https://github.com/FerretDB/github-actions): This repository houses our shared GitHub Actions for FerretDB, dance, and other repositories.
+* [dance (stub)](https://github.com/FerretDB/dance): This repository is a FerretDB integration testing tool.
+
+For those taking their first steps in contributing to an open source project.
+Please take a look at this post on [how to contribute to open source software](https://www.ferretdb.io/how-to-contribute-to-open-source-2022/).
+
+## Get Started
+
+You don’t have to be a developer to contribute, you can even get started by helping us improve this documentation.
+Here's some ideas to get you started:
+
+* Simply fork the FerretDB project you want to contribute to.
+Read the [CONTRIBUTING.md](https://github.com/FerretDB/FerretDB/blob/main/CONTRIBUTING.md) and [CODE_OF_CONDUCT.md](https://github.com/FerretDB/FerretDB/blob/main/CODE_OF_CONDUCT.md) files to know more about contributing to the project.
```

Comment:
The idea was to link to https://github.com/FerretDB/FerretDB/blob/main/CONTRIBUTING.md for FerretDB, to https://github.com/FerretDB/dance/blob/main/CONTRIBUTING.md for dance, and to a separate CONTRIBUTING.md in github-actions repo afte FerretDB/github-actions#96